source/managing_account/02_managing_api_keys/managing_api_keys.rst

```diff
@@ -0,0 +1,53 @@
+Managing API keys
+=================
+
+After authenticating to a portal, you can generate API keys and manage them from the API keys tab of their account.
+
+API keys are bound to a specific domain. They only work on the domain on which they were created.
+
+These API keys allow you to use the following APIs: ODS Search API v1 and v2, OData, WFS, and CSW. For more info, see :doc:`Using APIs </exploring_catalog_and_datasets/04_getting_involved/using_api>`.
+
+.. admonition:: Important
+   :class: important
+
+   API keys inherit all permissions granted to the user for which it has been created.
+   That's why users should be cautious with how they use API keys.
```

Comment:
It seems to me that this documentation is for users who create their own API keys, yet speaking of "users" seems confusing: shouldn't it be "you should be cautious (...)"?

Reply:
Totally agree with this. We should address directly to the user.